app/controllers/bookable_slots_controller.rb

```diff
@@ -1,7 +1,30 @@
 class BookableSlotsController < ApplicationController
+  def index
+    slots = BookableSlot
+            .without_holidays
+            .within_date_range(start_at, end_at)
+    slots = slots.where(guider: current_user) if scoped_to_me?
```

Comment:
I wonder if this would be nicer defined as a scope on `BookableSlot` instead? It's not a huge amount of code so not overly concerned.